lighthouse-core/module/index.js

```diff
@@ -18,9 +18,9 @@
 'use strict';
 
 const semver = require('semver');
-const log = require('../src/lib/log.js');
-const ChromeProtocol = require('../src/lib/drivers/cri.js');
-const lighthouse = require('../src/lighthouse');
+const Runner = require('../runner');
```

Comment:
As a bystander I'm not totally clear on the separation between module/index, runner and core. 
How would you describe it?